### PR TITLE
Scaffold index

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: front-page.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+          if (!empty($work['image_id'])) {
+            $work_image = wp_get_attachment_image($work['image_id'], 'home-work', array('class'=>'home-work-image'));
+          }
+
+          $work_title = get_the_title($work['work_id']);
+          $work_cats = get_the_category($work['work_id']);
+
+?>
+
+          <div class="home-work-item">
```
Comment (varffs) [inline front-page.php]: I think the whole element should be linked

File: front-page.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+?>
+        <article class="u-alert grid-item item-s-12"><?php _e('Sorry, no posts matched your criteria :{'); ?></article>
+<?php
+} ?>
+
+      </div>
+    </div>
+  </section>
+
+  <?php get_template_part('partials/pagination'); ?>
```
Comment (varffs) [inline front-page.php]: This shouldn't be here

File: lib/meta-boxes.php [hunk 3]
```diff
@@ -39,6 +39,48 @@ function igv_cmb_metaboxes() {
...
+   'object_types'  => array( 'page', ), // Post type
+   'show_on'      => array( 'key' => 'id', 'value' => $home_page->ID),
+  ) );
+
+  $home_work_group = $home_meta->add_field( array(
+    'id'         => $prefix . 'home_work_group',
+    'desc'       => esc_html__( 'Display and position Work thumbnails', 'cmb2' ),
+    'type'       => 'group',
+    'options'     => array(
+      'group_title'   => __( 'Work {#}', 'cmb2' ), // since version 1.1.4, {#} gets replaced by row number
```
Comment (varffs) [inline lib/meta-boxes.php:57]: Is Work the correct name?

**Review by varffs — CHANGES_REQUESTED** [review]

File: lib/meta-boxes.php (hunk 3 above)
Comment (mikeyrayvon) [reply lib/meta-boxes.php:57]: any suggestions?

Comment (mikeyrayvon) [reply lib/meta-boxes.php:57]: Its the name of the post type so i figure match it to that

Comment (mikeyrayvon) [reply lib/meta-boxes.php:57]: `Project` ?

**Review by varffs — APPROVED** [review]

**Pull request merged**